### ISSUE TITLE
Xamarin Forms - System.BadImageFormatException was thrown: Could not resolve field token 0x04000219

### ISSUE THREAD

**Issue by CliffCawley** [keenlabs/keen-sdk-net#33]
Although I am now having another problem when trying to add an event:

```
			var prjSettings = new ProjectSettingsProvider("~REMOVED~", writeKey: "~REMOVED~");
			var keenClient = new KeenClient(prjSettings);
			keenClient.AddGlobalProperty("clienttype", "mobile");

						// Build an event object
			var aPurchase = new
			{
				category = "magical animals",
				username = "hagrid",
				price = 7.13,
				payment_type = "information",
				animal_type = "norwegian ridgeback dragon",
				user_ip = "8.8.8.8",
				ua = "Mozilla/4.0 (compatible; MSIE 5.0; Windows NT; DigExt; .NET CLR 1.0.3705)"
			};

			// send the event
 			keenClient.AddEvent("purchases", aPurchase);
```

the AddEvent call asserts with: 
```
System.BadImageFormatException was thrown: Could not resolve field token 0x04000219
```

**Comment by elof** [comment]
Hi there. Can you send the info in this inssue along with us a stack trace and repro steps to support via team@keen.io? We will take a look and respond over there. 

Generally speaking emailing support or jumping in at http://keen.chat will get you to the right people and will have a faster response time.

**Comment by CliffCawley** [comment]
Hey, I've spent some more time on this and I think it was because I had to hack using the 0.3.13 library, which meant the dependencies weren't properly updated. This meant I was using slightly older versions of the packages (as I already have them in my project).

Now that you've released 0.3.13 and I updated it seems to be working correctly.

I'll close this :)

**Comment by elof** [comment]
Thanks for the note @CliffCawley. Hit us up if you have any other questions.
